include/can/core/messages.hpp

```diff
@@ -720,6 +720,40 @@ struct SetBrushedMotorPwmRequest
         -> bool = default;
 };
 
+struct AddBrushedLinearMoveRequest
+    : BaseMessage<MessageId::add_brushed_linear_move_request> {
+    uint8_t group_id;
+    uint8_t seq_id;
+    brushed_timer_ticks duration;
```

Comment:
why do we need `duration` and `duty_cycle`? those values aren't used, right?

Reply:
duration currently isn't used by any of the gripper moves, we have a ticket in our queue to use duration to trigger a timeout so moves don't get stuck forever.
and on the monorepo side all the gripper messages have the same payload type for consistency there, so duty cycle is a hold over, I may use this in the future as a duty_cycle MAX value